corehq/apps/users/util.py

```diff
@@ -270,6 +271,8 @@ def update_device_meta(user, device_id, commcare_version=None, device_app_meta=N
                 device_id,
                 commcare_version=commcare_version,
                 device_app_meta=device_app_meta,
+                fcm_token=fcm_token,
+                fcm_token_timestamp=fcm_token_timestamp
```

Comment:
Confirming that this is the only use of this method that we need to update?

Reply:
Confirmed !